scripts/serve_dev.ts

```diff
@@ -24,6 +24,15 @@ if (process.env.DEV_LOGGING) {
   app.use(morgan('ϟ :method :url :status :response-time ms'));
 }
 
+// this global is set when this file is run with `nyc`
+if ((global as any).__coverage__) {
```

Comment:
There are better ways to do this: https://stackoverflow.com/questions/38906359/create-a-global-variable-in-typescript